settings.php

```diff
@@ -15,16 +15,6 @@
 		max-width: none;
 		white-space: nowrap;
 	}
-	@-webkit-keyframes Pulse{
-		from {color:#630030;-webkit-text-shadow:0 0 9px #333;}
-		50% {color:#e33100;-webkit-text-shadow:0 0 18px #e33100;}
-		to {color:#630030;-webkit-text-shadow:0 0 9px #333;}
```

Comment:
Please have a look in #518.
There, in `footer.php` was the almosts equal pulse animation as it is here in `settings.php`.
The `text-shadow`-animation there (and in here, too) was/(is) written to work only on `-webkit-`browsers, but not on others (like Firefox or Edge).
The overhauled rule, which I added to `pi-hole.css`, should work across all browsers.
And because a smaller text-shadow-blur-radius is used, the animation even needs less client's CPU performance, and not looks really different as before.